### DISCUSSION TITLE
Can anyone please tell me how we can build the SLF4j - 1.8.0-beta2, Which version of maven it is using.

### DISCUSSION THREAD

**Discussion by JagatP-logins** [qos-ch/slf4j#436] [Q&A]
I am facing issues while building the project **SLF4j - 1.8.0-beta2**, It is using jdk6 but maven version in not specified anywhere.

Please help me if anyone aware of it.

Thanks in advance

**Comment by ceki** [comment]
SLF4J version 1.8.0-beta2 has been deprecated and replaced by the 2.0.x series.
